settings.gradle.kts

```diff
@@ -14,6 +14,8 @@ include(":wire-gradle-plugin-playground")
 include(":wire-grpc-tests")
 include(":wire-protoc-compatibility-tests")
 
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
This is not needed. This is for `projects.whevs` references.

Reply:
I'll do in follow up